index.html

```diff
@@ -18,7 +18,7 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
       </header>
       <form id="survey-form">
```

Comment:
Good job using HTML5 semantic tags (e.g., header, nav) as an optional suggestion I would suggest using of `<section> <section/> `tags and include the contents inside section sections like, section with classname "main" , "aside", "footer" etc. 😊😊